solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -236,16 +240,9 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
     {
         uint256 reserveCount = __reserveTokens.length;
         for (uint256 i = 0; i < reserveCount; ++i) {
-            IERC20 reserveToken = __reserveTokens[i];
-
-            uint256 amount;
-            if (reserveToken == NATIVE_TOKEN_ADDRESS) {
-                amount = address(this).balance;
-            } else {
-                amount = reserveToken.balanceOf(address(this));
-            }
+            IReserveToken reserveToken = __reserveTokens[i];
 
-            safeTransfer(reserveToken, _newConverter, amount);
+            reserveToken.safeTransfer(_newConverter, reserveToken.balanceOf(address(this)));
```

Comment:
Can use `safeTransferAll` here?

Reply:
Please check @yudilevi's comment. We decided to remove it.